modules/context/context.go

```diff
@@ -86,7 +69,7 @@ func (ctx *Context) Close() error {
 	return err
 }
 
-// TrHTMLEscapeArgs runs Tr but pre-escapes all arguments with html.EscapeString.
+// TrHTMLEscapeArgs runs "locale.Tr()" but pre-escapes all arguments with html.EscapeString.
```

Comment:
It calls `ctx.Tr`, not `locale.Tr`. So the old comment was correct?

Reply:
`ctx.Tr` is `ctx.Locale.Tr`.

```suggestion
// TrHTMLEscapeArgs runs ".Locale.Tr()" but pre-escapes all arguments with html.EscapeString.
```

Reply:
Does the new change look good to you?